appdata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.geogebra.GeoGebra.desktop</id>
+  <name>GeoGebra</name>
+  <developer_name>GeoGebra</developer_name>
+  <summary>Dynamic mathematics software</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
Ah didn't know. I thought it was fine to have a flatpak when I saw rpm, deb, and other installers available. OK you can close this PR then.